rts/lra/lra-coordinator/src/main/java/io/narayana/lra/coordinator/domain/model/LRARecord.java

```diff
@@ -320,7 +320,19 @@ private int tryDoEnd(boolean compensate) {
 
                 httpStatus = response.getStatus();
 
-                accepted = httpStatus == Response.Status.ACCEPTED.getStatusCode();
+                ParticipantStatus participantStatus = null;
```

Comment:
@mmusgrov yes, this one other bug I discovered when running newly added TCK. I will raise a separate JIRA for it then.

Reply:
@mmusgrov as I'm looking at these code changes in `LRARecord` they fix the issue reported as the JBTM-3156. I think they should be fine to be accepted for the PR.
Please, what is the functional trouble with these code changes from your perspective?

Reply:
@mmusgrov @xstefank I can see there is a new issue on this in the LRA spec https://github.com/eclipse/microprofile-lra/issues/173 and the https://issues.jboss.org/browse/JBTM-3156 was rejected. I'm not sure I do understand what is the intended behaviour. Is it really in way that the `Compensating` and `Completing` state is really wrong (not `accepted`) here? I probably don't understand the spec at this place and I would need more elaboration on this.

Reply:
@ochaloup Yes, the new section on "Eventual Consistency" introduced the ambiguity.

Reply:
@mmusgrov @ochaloup however, can we please merge this PR in its current form and adjust eclipse/microprofile-lra#173 afterwards. It will definitely require TCK changes so additional PR on top of this one will be clear distinction IMO.

Reply:
yes, I'm fine with that approach

Reply:
This conversation hasn't been resolved in you latest commit

Reply:
@mmusgrov this is waiting for your input, can we go with these changes and adjust if necessary after eclipse/microprofile-lra#173 is resolved?

Reply:
There are no MP-LRA tests that would exercise this change of behaviour you have added so I don't accept it should be in this PR. 202 Accepted is the way that participants report in progress compensations (it has been that way since before the sandbox). If the outcome of PR 173 is not to use 202 then yes we would need to modify the implementation but let's not make changes before they are required.